SRC/css-grids/css-grids.css

```diff
@@ -0,0 +1,59 @@
+.mainContainer{
+    background-image: url(https://cdn.cjr.org/wp-content/uploads/2019/07/AdobeStock_100000042-e1563305717660-686x371.jpeg);
```

Comment:
so a smaller background will always repeat and make tiles. If that's what you wanted, great! if not, there are properties that will help cover your element. look up the background-repeat & background-size properties. tons of values to use for those! 
give this a try and see if you like it:
`background-repeat: no-repeat;`
 `background-size: cover;`